Signal/src/ViewControllers/SignalAttachment.swift

```diff
@@ -279,16 +279,6 @@ class SignalAttachment: NSObject {
         return MIMETypeUtil.supportedAudioUTITypes()
     }
 
-    public class var textUTISet: Set<String> {
-        return [
-            kUTTypeText as String,
-            kUTTypePlainText as String,
-            kUTTypeUTF8PlainText as String,
-            kUTTypeUTF16PlainText as String,
-            kUTTypeURL as String
```

Comment:
IIRC pasting a URL was considered an attachment before, just wanted to make sure you tested that pasting attachments still worked with the new "conforms" check.